### PR TITLE
[s] Bullies a certain small-penised individual

### PR DESCRIPTION
**Pull request by PKPenguin321**
If this isn't grudge coding I don't know what is, but hey it's april fools so really it's just a prank bro

### CODE REVIEW

**Comment by PKPenguin321** [comment]
@kevinz000

**Comment by ChangelingRain** [comment]
put the kekhi on the table

**Comment by Xhuis** [comment]
this is the best thing i've ever seen

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -183,7 +183,11 @@
 		if(!isturf(picked))
 			return
 		H.forceMove(picked)
-		H.rad_act(rad_amount)
+		if(H.ckey == "kevinz000")
+			H.rad_act(99999999999999999999999)
```
Comment (Xhuis) [inline code/modules/clothing/suits/armor.dm]: rofl

**Comment by optimumtact** [comment]
This is violation of the player reference policy and for those reasons will be closed

**Comment by PKPenguin321** [comment]
b-but https://github.com/tgstation/-tg-station/pull/16344 has like 40x the player references

**Comment by Bawhoppen** [comment]
>implying oranges actually has any real reason to exist in tg community

**Comment by kevinz000** [comment]
B-bully

**Comment by Jordie0608** [comment]
Even for april fools, this is bad

**Comment by optimumtact** [comment]
I warned you

**Comment by PKPenguin321** [comment]
fuck

**Comment by Bawhoppen** [comment]
oh sweet I didn't lose my position cause kevinz found out on his own

**Pull request closed without merging**